### DISCUSSION TITLE
[Feature] Why is php-cli mode not supported

### DISCUSSION THREAD

**Discussion by tangzuqiang** [apache/skywalking#12854] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Why is php-cli mode not supported

### Use case

Please support cli mode for link tracing of scripts and scheduled tasks

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by tangzuqiang** [comment]
php currently supports PHP-FPM mode, link tracing is not available in command line mode, can it be supported in later versions?

**Comment by jmjoy** [comment]
SkyWalking is designed for request tracing, and it relies on request context in order to perform link tracking, like swoole.
